appveyor.yml

```diff
@@ -0,0 +1,9 @@
+version: '4.5.4-SNAPSHOT+AppVeyor.{build}'
```

Comment:
:bulb: This will quickly become incorrect. You can improve on this by moving the version handling to the build script [like this](https://github.com/Microsoft/CodeContracts/blob/master/appveyor.yml#L5). To determine the specific build number for the call to `Update-AppveyorBuild`, you can extract it from the **pom.xml** file.